spec/robots/accession/technical_metadata_spec.rb

```diff
@@ -47,6 +47,7 @@
                                         {
                                           externalIdentifier: '222-1',
                                           label: 'folder1PuSu/story1u.txt',
+                                          filename: 'folder1PuSu/story1u.txt',
```

Comment:
@jcoyne is this what you'd expect here? is it OK if the filename has path info in it?

Reply:
I think so